app/server/load-data.js

```diff
@@ -47,14 +49,17 @@ export function loadErrorData(error, config) {
 
 // FIXME: Convert this to async/await
 export function loadData(pageType, params, config, client, { host, next, domainSlug }) {
+  const publisherAttributes = getPublisherAttributes();
   function _loadData() {
     switch (pageType) {
       case PAGE_TYPE.HOME_PAGE:
         return loadHomePageData(client, config, params.collectionSlug);
       case PAGE_TYPE.HOME_PREVIEW:
         return loadHomePageData(client, config);
       case PAGE_TYPE.SECTION_PAGE:
-        return loadSectionPageData(client, params.sectionId, config);
+        return loadSectionPageData(client, params.sectionId, config, publisherAttributes);
+      case PAGE_TYPE.COLLECTION_PAGE:
+        return loadCollectionPageData(client, params.collectionSlug, config);
```

Comment:
can we use common util function to fetch collection data for home and collection page ?